src/EnergyPlus/DXCoils.cc

```diff
@@ -14292,7 +14292,7 @@ Label50: ;
 		}
 		return SHR;
 	}
-
+	
```

Comment:
[CodeMaid](http://www.codemaid.net/) works great for my install of VS 2012, but doesn't load for 2013 Express. Maybe I just don't have something setup correctly. If anyone tries it and gets it to work, feel free to pass it along (in a separate thread).